test/aws/osml/model_runner/tile_worker/test_variable_overlap_tiling_strategy.py

```diff
@@ -53,28 +71,40 @@ def test_compute_regions_roi(self):
             ]
 
     def test_compute_regions_tiny_image(self):
+        """
+        Test that regions are correctly computed for a tiny image where the
+        nominal region size is larger than the image itself.
+        """
         from aws.osml.model_runner.tile_worker import VariableOverlapTilingStrategy
 
         tiling_strategy = VariableOverlapTilingStrategy()
 
+        # Define tiling parameters
         nominal_region_size = (10000, 10000)
         overlap = (100, 100)
         tile_size = (4096, 4096)
 
-        # Check regions generated from an image that has a dimension smaller than the fixed tile size
+        # Compute regions for a tiny image
         regions = tiling_strategy.compute_regions(((0, 0), (12000, 2000)), nominal_region_size, tile_size, overlap)
+
+        # Verify the number of computed regions
         assert len(regions) == 2
+
+        # Verify that all computed regions match expected results
```

Comment:
not sure these 2 comments add any additional insight beyond the actual assertions.